docs/source/bricks.rst

```diff
@@ -750,3 +750,26 @@ files to an S3 bucket.
 
   upload_staged_files()
 
+``stage_for_datasaur``
+--------------------------
+Formats a list of Text elements as input to Datasaur.
```

Comment:
Text -> `Text`

Reply:
Can you also mention in the description that this is specific to token based tasks from Datasaur?